src/main/java/org/soen343/models/parameters/SimulationParameters.java

```diff
@@ -1,7 +1,7 @@
 package org.soen343.models.parameters;
 
 public class SimulationParameters {
-    private static DateTime dateTime;
+    public static DateTime dateTime;
```

Comment:
This should be private.

To access the simulation parameters. Use Model.getSimulationParameters().getDateTime().. ...